### ISSUE TITLE
Wrap Filter Options in Fieldset with Legend

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#3131]
### Descriptive summary

In the dashboard for Collections and Works in Hyrax 2.1.0 RC3, there are filter options that are dropdown button for options like "Visibility" or "Status." Each individual dropdown should be contained within a `<fieldset>` with a `<legend>` .

### Rationale

Using `<fieldset>` and `<legend>` ensures that the text description is read to screen reader users when the grouping is navigated to when exploring the interface. The legend text is generally read for each control in the fieldset, so the legend text should be brief and descriptive.

### Expected behavior

Groupings of form controls should be wrapped in a `<fieldset>` and `<legend>`.

### Actual behavior

Bootstrap `btn-group` is being used to organize the controls, but the CSS class is not usable for screen readers.

**Comment by mtribone** [comment]
![screen shot 2018-06-26 at 12 57 22 pm](https://user-images.githubusercontent.com/4163828/41927531-851a7e2a-7940-11e8-9cac-ba4edb8aaf04.png)
